support-frontend/app/admin/settings/SettingsProvider.scala

```diff
@@ -2,13 +2,12 @@ package admin.settings
 
 import java.util.concurrent.Executors
 import java.util.concurrent.atomic.AtomicReference
-import admin.settings.SettingsProvider._
+import admin.settings.SettingsProvider.SettingsUpdate
```

Comment:
this had to be more specific because importing everything from SettingsProvider also pulled in the `logger`.  That causes a cyclical import issue.